### DISCUSSION TITLE
Testing Realtime Robustness of ROS in Autonomous Driving (Project 220)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#52] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 220](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Testing%20Realtime%20Robustness%20of%20ROS%20in%20Autonomous%20Driving)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Sukumar4015** [comment]
Hi, 
I am a student at IIT ROORKEE and interested in working and collaborating on this project (Testing Realtime Robustness of ROS in Autonomous Driving (Project 220)).

**Comment by shantanuparabumd** [comment]
Hi,
I am Shantanu Parab a student at University of Maryland and interested in working and collaborating on this project (Testing Realtime Robustness of ROS in Autonomous Driving (Project 220)).

**Comment by floriandev70** [comment]
Hi this is Florian MSc. CS graduate from TU Dortmund, just started working on this project. Open for collaboration.
